packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2813,15 +2827,12 @@ export class ContainerRuntime
 			// Process the previous bunch of messages.
 			const sendBunchedMessages = (): void => {
```

Comment:
Since you are touching it, maybe worth renaming sendBunchedMessages to processBunchedMessages?